diff --git a/lib/PlugsManager.vala b/lib/PlugsManager.vala
index d19be705..8c9fba54 100644
--- a/lib/PlugsManager.vala
+++ b/lib/PlugsManager.vala
@@ -37,22 +37,14 @@ public class Switchboard.PlugsManager : GLib.Object {
     private PlugsManager () {
         plugs = new Gee.LinkedList<Switchboard.Plug> ();
 
-        string[] prefixes = {
-            "/usr",
-            "/usr/local"
-        };
-        string[] plugin_paths = {};
-        foreach (unowned string prefix in prefixes) {
-            var plugin_dir = Path.build_filename (prefix, Build.LIB_DIR, "switchboard-3") + "/";
-            if (FileUtils.test (plugin_dir, FileTest.EXISTS)) {
-                debug ("Found plugin directory: %s", plugin_dir);
-                plugin_paths += plugin_dir ;
-            }
+        var plugs_path = Environment.get_variable ("SWITCHBOARD_PLUGS_PATH");
+        if (plugs_path != null) {
+            debug ("SWITCHBOARD_PLUGS_PATH set to %s", plugs_path);
+        } else {
+            critical ("SWITCHBOARD_PLUGS_PATH not set");
         }
 
-        foreach (unowned string path in plugin_paths) {
-            find_plugins (File.new_for_path (path));
-        }
+        find_plugins (File.new_for_path (plugs_path));
     }
 
     private void load (string path) {
