diff --git a/src/PopoverWidget.vala b/src/PopoverWidget.vala
index 6fa7a95..119c99a 100644
--- a/src/PopoverWidget.vala
+++ b/src/PopoverWidget.vala
@@ -161,7 +161,7 @@ public class Keyboard.Widgets.PopoverWidget : Gtk.Box {
     }
 
     private void show_keyboard_map () {
-        string command = "gkbd-keyboard-display \"--layout=" + get_current_with_variant () + "\"";
+        string command = "@gkbd_keyboard_display@ \"--layout=" + get_current_with_variant () + "\"";
 
         try {
             AppInfo.create_from_commandline (command, null, AppInfoCreateFlags.NONE).launch (null, null);